tests/foreman/ui/test_host.py

```diff
@@ -1670,6 +1670,55 @@ def test_global_registration_with_gpg_repo_and_default_package(
     assert repo_url in result.stdout
 
 
+@pytest.mark.tier2
+def test_global_registration_upgrade_subscription_manager(
+    session, module_activation_key, module_os, module_proxy, rhel7_contenthost
+):
+    """Host registration form produces a correct registration command and
+    subscription-manager can be updated from ABI or custom repository before
+    registration is completed.
+
+    :id: b7a44f32-90b2-4fd6-b65b-5a3d2a5c5deb
+
+    :expectedresults: Host is successfully registered, repo is enabled
+        on advanced tab and subscription-manager is updated.
+
+    :CaseLevel: Integration
+
+    :steps:
+        1. Create activation-key
+        2. Open the global registration form, add repo and activation key
+        3. Add 'subscription-manager' to install packages field
+        4. Check subscription-manager was installed from repo_name
+
+    :parametrized: yes
```

Comment:
iirc `parametrized` token should mark tests that are executed several times in different contexts (based on parameters) during the test run. If you accept tstrych's suggestion above, then `:parametrized: yes` should stay, but otherwise I think it doesn't apply

Reply:
ACK, thank you.